abis_mapping/templates/survey_site_data/mapping.py

```diff
@@ -147,10 +151,17 @@ def extract_geometry_defaults(
         # Context manager for row streaming
         with resource.open() as r:
             # Create empty dictionary to hold mapping values
-            result = {}
+            result: dict[str, str] = {}
             for row in r.row_stream:
                 # Extract values
-                site_id: str = row["siteID"]
+                site_id = _string_or_none.validate_python(row["siteID"])
```

Comment:
The output of the [frictionless string field](https://github.com/frictionlessdata/frictionless-py/blob/main/frictionless/fields/string.py#L87) is `str | None` .  Do we need to validate again?